ui/packages/consul-ui/translations/routes/en-us.yaml

```
@@ -213,12 +213,19 @@ dc:
                 <a href="{CONSUL_DOCS_URL}/connect/registration/service-registration#upstream-configuration-reference" target="_blank" rel="noopener noreferrer">Read the documentation</a>
               </p>
           acls-disabled:
-            header: Enable ACLs
-            body: This connect-native service may have dependencies, but Consul isn't aware of them when ACLs are disabled. Enable ACLs to make this view more useful.
+            header: All service-to-service connections are allowed
```

@jnwright , @johncowen : Should this be consistent with the other "Restrict which services can connect" headers?

```suggestion
            header: Restrict which services can connect
```

---

Hm, I'm not sure. The title you and I chose during our sessions is the one John wrote. I remember us wanting to differentiate it for a reason, but do you think it should be consistent?

---

I vaguely recall such a conversation, but not what the outcome/decision was!

The two messages are basically the same, just with a different focus. "Restrict which services can connect" focuses on what the user should do. "All service-to-service connections are allowed" focuses on communicating current behavior. I'm not sure why we'd frame the same message in two different ways depending on whether ACLs are disabled vs enabled with default allow. ... Maybe we did have a reason when we last spoke about it, but I can't remember if it we did!

---

I'm fine with changing it to be the same title ¯\_(ツ)_/¯

---

What was the outcome here? Shall I go ahead and commit the suggestion or leave as is?

---

yeah, I say we go ahead and change it! And that'll wrap this up 😎

---

Perfect, k thats done, just needs a 👍 /approval on this one then and we can merge